<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FlightDialog</class>
 <widget class="QDialog" name="FlightDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>467</width>
    <height>516</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLineEdit" name="leditNumber">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Number</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leditNumber</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLineEdit" name="leditDestination">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="font">
      <font>
       <pointsize>8</pointsize>
      </font>
     </property>
     <property name="text">
      <string>Destination</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>leditDestination</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QDateTimeEdit" name="dtimeEditDeparture">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="maximumDate">
      <date>
       <year>2027</year>
       <month>1</month>
       <day>1</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2022</year>
       <month>5</month>
       <day>20</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Departure</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>dtimeEditDeparture</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0">
    <widget class="QDateTimeEdit" name="dtimeEditArrival">
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="minimumDateTime">
      <datetime>
       <hour>4</hour>
       <minute>0</minute>
       <second>0</second>
       <year>2022</year>
       <month>5</month>
       <day>20</day>
      </datetime>
     </property>
     <property name="maximumDate">
      <date>
       <year>2027</year>
       <month>1</month>
       <day>3</day>
      </date>
     </property>
     <property name="minimumDate">
      <date>
       <year>2022</year>
       <month>5</month>
       <day>20</day>
      </date>
     </property>
     <property name="calendarPopup">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Arrival</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="buddy">
      <cstring>dtimeEditArrival</cstring>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>FlightDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>FlightDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
